src/app/containers/PageHandlers/withData.jsx

```diff
@@ -1,39 +1,65 @@
-import React from 'react';
+import React, { useContext } from 'react';
 import { shape, element } from 'prop-types';
 import articlePropTypes from '../../models/propTypes/article';
 import ErrorMain from '../ErrorMain';
 import nodeLogger from '../../helpers/logger.node';
 import deepGet from '../../helpers/json/deepGet';
+import { ServiceContext } from '../../contexts/ServiceContext';
+import idSanitiser from '../../lib/utilities/idSanitiser';
 
 const logger = nodeLogger(__filename);
 
-// checks for data and status keys, setting default status if not found
+/* used to remove - & _ from locale and service language for comparison
+eg: transforms en-GB, en_gb to engb, etc.
+TODO: move this to a util ? not sure where exactly
```

Comment:
We have utilities in this directory: https://github.com/bbc/simorgh/tree/latest/src/app/lib/utilities